src/Schema/Events/Click.ts

```diff
@@ -1742,4 +1742,61 @@ export interface ClickedVerifiedRepresentative {
   context_page_owner_id: string
   destination_page_owner_type: PageOwnerType
   destination_page_owner_id: string
+}
+
+/**
+ * A user clicks one of the related categories(genes) in the artist about tab
+ *
+ *  This schema describes events sent to Segment from [[clickedRelatedCategory]]
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "clickedRelatedCategory",
+ *    context_module : "Young British Artists",
+ *    context_screen_owner_type: "Artwork",
+ *    context_screen_owner_id: "58de681f275b2464fcdde097",
+ *    context_screen_owner_slug: "damien-hirst",
+ *    destination_screen_owner_type: "Gene"
+ *    destination_screen_owner_id: "58de681f275b2464fcdde097"
+ *    subject: ""
+ *  }
+ * ```
+ */
+export interface ClickedRelatedCategory {
```

Comment:
We use the word `gene` everywhere, so isn't better to use the same naming for the events too to keep the terminology consistent?

Reply:
Good call, I'll fix that